tracing.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const url = require('url');
+
+const tracing = require("signalfx-tracing");
+const opentracing = require("opentracing");
+const NoopTracer = require("signalfx-tracing/src/noop/tracer");
+
+const _defaultEndpointPath = '/v1/trace';
+
+// must init tracer here before other libraries are imported in order to apply
+// auto-instrumentation patches.
+let _tracer;
+
+if ((process.env.SIGNALFX_TRACING_ENABLED || "true").toLowerCase() == "false") {
+  _tracer = new NoopTracer();
+} else {
+  const options = {
+    service:
+      process.env.SIGNALFX_SERVICE_NAME || process.env.AWS_LAMBDA_FUNCTION_NAME,
+  };
+  if (process.env.SIGNALFX_ENDPOINT_URL) {
+    const parsed = url.parse(process.env.SIGNALFX_ENDPOINT_URL); 
+    console.log(parsed)
+    if (parsed.pathname === "/") {
```

Comment:
should we also confirm that no metrics or ingest url env vars are set?  users may have their own proxy rules, though it's unlikely

Reply:
Added a warning message.